cDevice_Build.sh

```diff
@@ -13,20 +13,6 @@ fi
 
 # Build To Archive into Artifactory
 
-echo "For ecdsa_c_device *******"
```

Comment:
Considering there is no manufacturer in PRI anymore, should we remove MANUFACTURER_TOOLKIT flag and instead update the code to reflect this behavior by default.